lib/maths/common/unittest/CLinearAlgebraTest.cc

```diff
@@ -1159,12 +1160,12 @@ BOOST_AUTO_TEST_CASE(testShims) {
     {
         BOOST_TEST_REQUIRE(TMatrix4(0.0) ==
                            maths::common::las::conformableZeroMatrix(vector1));
-        BOOST_TEST_REQUIRE(TMatrix(4, 0.0) ==
-                           maths::common::las::conformableZeroMatrix(vector2));
-        BOOST_TEST_REQUIRE(TDenseMatrix::Zero(4, 4) ==
-                           maths::common::las::conformableZeroMatrix(vector3));
-        BOOST_TEST_REQUIRE(TDenseMatrix::Zero(4, 4) ==
-                           maths::common::las::conformableZeroMatrix(vector4));
+        BOOST_TEST_REQUIRE((TMatrix(4, 0.0) ==
```

Comment:
Workaround for incompatibility between BOSST_TEST_REQUIRE macro and the template instantiation of these Eigen types